cmd/ll2go/main.go

```diff
@@ -511,6 +511,14 @@ func (d *decompiler) value(v value.Value) ast.Expr {
 	}
 }
 
+func (d *decompiler) int(i int64) ast.Expr {
```

Comment:
I did not know you could use keywords (such as `int`) as method names in Go. Does this compile?

Even so, I would prefer if we renamed the `int` method to `intLit` and added a comment along the lines of

```go
// intLit converts the given integer literal into a corresponding Go expression.
func (d *decompiler) intLit(i int64) ast.Expr {
...
}
```